internal/validitywindow/handler_test.go

```diff
@@ -124,11 +123,14 @@ func generateBlockChain(n int, containersPerBlock int) map[uint64]ExecutionBlock
 	for i := 1; i < n; i++ {
 		containers := make([]int64, containersPerBlock)
 		for j := 0; j < containersPerBlock; j++ {
-			containers[j] = rand.Int63n(int64(n)) //nolint:gosec
+			containers[j] = int64(j + 1)
+			if containers[j] < int64(i) {
+				containers[j] = int64(i) + 1
+			}
```

Comment:
Will this result in duplicates? I believe to make this data correct we need to create test containers that depend on more than just the expiry time.